L/libjulia/libjulia@1.6/build_tarballs.jl

```diff
@@ -0,0 +1,2 @@
+include("../common.jl")
+build_julia(v"1.6-dev"; gitref = "d474c98667db0bf4832e4eeb7beb0e8cfc8b7481")
```

Comment:
Regarding the version, perhaps we should imitate what Julia itself does, and use something like `1.6.0-DEV.1390` as version. (I'd not bother to replicate the code for computing the integer '1390' but rather copy it manually from a build of the given git commit).

Sounds OK?

Reply:
You can also use `contrib/commit-name.sh` to map from a git sha to a julia build name.

Reply:
Sure, but I don't see how that helps, as the build script which has access to this is only run inside the `build_tarballs` function, which already requires the version as an argument?